server.js

```diff
@@ -1,33 +1,37 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+const port = process.env.PORT || 8080;
+const app = express();
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+// Middlewares to enable cors and json body parsing
+app.use(cors());
+app.use(bodyParser.json());
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+// Routes
+app.get("/", (req, res) => {
+  res.send("Hello world");
+});
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+app.get("/books", (req, res) => {
+  res.json(booksData);
+});
+
+app.get("/books/:id", (req, res) => {
+  const bookId = req.params.id;
+  const book = booksData.find(item => item.bookID.toString() === bookId);
+  res.json(book);
+});
+
+app.get("/language/:language", (req, res) => {
```

Comment:
It would be nice if this was part of the `/books` endpoint (and it'd be more RESTful) - then make use of query params to filter on language. So you could have `/books` to return all books and then `/books?language=en` for english books, etc.